### ISSUE TITLE
Paste as Value (ctrl+shift+v) feature does not paste empty cells in Kendo Spreadsheet

### ISSUE THREAD

**Issue by nenchef** [telerik/kendo-ui-core#5319]
### Bug report

Paste as Value (ctrl+shift+v) feature does not paste empty cells in Kendo Spreadsheet.

### Reproduction of the problem
1. Open https://demos.telerik.com/kendo-ui/spreadsheet/index

2. Copy from F3:F15

3. Paste as value (ctrl+shift+v) in H3

4. See it pasted only non-empty cells

### Current behavior
_(optional)_
The Spreadsheet pastes the next value, in a cell that should be empty.

### Expected/desired behavior
The spreadsheet should paste empty values as well

### Environment

* **Kendo UI version:** 2019.3.917
* **Browser:** [all]

**Comment by Dimitar-Goshev** [comment]
@nenchef @mishoo Paste as Values (Ctrl + Shift +V) is a browser specific feature, which currently works only in Chrome and Firefox. Previously it was working only in Chrome as I commented in [Allow the Spreadsheet Copy - Paste of only values (no formatting) from Excel](https://github.com/telerik/kendo-ui-core/issues/3336#issuecomment-319302265), when testing it in 2017. This feature was implemented as a [Knowledge Base article, handling the paste event](https://docs.telerik.com/kendo-ui/knowledge-base/spreadsheet-paste-only-values).

@mishoo The fix introduces new issues in Firefox and Chrome, where Ctrl + Shift + V works.
There are multiple empty rows inserted on pasting only values in this scenario:
1. Go to Spreadsheet Basic usage demo on staging.
1. Select range A1:G15 and copy it.
1. Click on cell A17 and paste with Ctrl+Shift+V

Current behavior:
Tip is in cell B46. Total Amount is in cell B49. There are 18 empty rows between Tip cell and the cells above, instead of only two rows. File attached below
[Workbook(1).xlsx](https://github.com/telerik/kendo-ui-core/files/3739398/Workbook.1.xlsx)
